variables.tf

```diff
@@ -254,6 +254,44 @@ variable "parquet_wandb_env" {
   default     = {}
 }
 
+##########################################
+# private link                           #
+##########################################
+
+## In order to support private link required min version 0.13.0 of operator-wandb chart
+
+variable "create_private_link" {
+  type        = bool
+  description = "Whether to create a private link service."
+  default     = false
+}
+
+variable "public_access" {
+  type        = bool
+  description = "Whether to create a public endpoint for wandb access."
+  default = true
+}
+
+variable "allowed_projects" {
+  type = map(number)
+  default = {
+    # "482878270665" = 4
```

Comment:
Is it possible to make this project name?

Reply:
Project name not supported we can only pass porject ID or project number, changed variable name now

variable "allowed_project_names" {
  type = map(number)
  default = {
    # "project_ID" = 4 
  }
  description = "A map of allowed project name where each key is a project number or ID and the value(4) is the connection limit."
}